sql/src/main/java/io/crate/execution/engine/collect/sources/SystemCollectSource.java

```diff
@@ -144,7 +144,8 @@ public SystemCollectSource(ClusterService clusterService,
                             supportMoveToStart,
                             records
                         )
-                    )
+                    ),
+            true
```

Comment:
Maybe add a comment that although most system sources are available in-memory, some are not and that's why we go for `true` ?